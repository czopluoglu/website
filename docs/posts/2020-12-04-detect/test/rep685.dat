01111011101000110101
11111111111010110000
01101011011111111110
01111101110001001111
10100100111000000011
10110101111100101011
10001100001000001000
10001010101111111101
10011110111010110010
11111110111111110011
11001111011011100111
11111111010000000000
00100110110101001000
01001110001111111011
00001010111011011111
10100000011010100001
11100010111111111010
11110101111011101101
01000101001010000001
11001101101010110111
11110001101111110110
10101000101101001011
11000110101100111111
11110110011111110011
11101101101111011011
11101111001011110101
10001000101110001000
11010111011111101010
11111110110000111110
11000001111011111011
01101110111101100010
00110001001110111111
01000111011110110111
00111111101111000110
11110100000101010111
11100100011011101100
00001101001101111110
00100100110110001110
00011001011110001110
11100101101110011111
11111011011100111011
00010100101010000011
00100100000001101010
10010111111111111111
11011100000010100001
11100110110010101110
01101111101101100001
11110111010001000010
01000101101000000001
00010000001111111011
00011110111110000111
00100000100010011000
01101101011010110010
10001111110100101011
01101011011000100110
00010000001011100110
01111001111101111110
01101110101011111110
10100100011101111011
01000111101110101111
11011100000111100010
11011001101011000110
00100101010111001111
01101111011111110110
01100000011110101111
01110101011111010111
11011101001111111001
11111111001110100011
11111011111110111111
10111011011100000010
00100011001110011010
00001101111110100111
00101011111110101110
01000010101101111011
10001110101000100111
00001101011011001100
10111100011110111111
01001001101011101111
10100100010010101111
01011001101111101111
11000001101110111111
10100011111111011111
01110100111101111111
01000101011010000001
01101111111110111110
01010011111010110010
01101001100010100111
11101010011010010110
01101111011010101110
00110001001011101010
00001011001011101100
10100101101000011000
10010101101000001111
11101011011001101111
10000000011101010111
01111110011011101110
00111000111110110011
01101111101111111111
01001001110100101101
00000000100010001111
00100101111111011010
00010000001001010111
10001001101010110110
10011000001000101100
11101111110110101101
00100101111100101101
11001100101111010110
10010111101000000111
10010101011101101110
10001001010110111011
11010101111010110111
10000011001010000010
00010011110010111110
01111111011111111111
10111101101111101010
11001111100000111111
01110011111110101110
01111001111111111111
00100101101111110101
10101001111000001010
11010001111100110111
11001001111010001011
00111110111110010101
01110010101110001111
11011110001110111110
10111111111101111011
11110110101100101001
00111101111001110111
11011000111011111011
10111011111000011111
10000001111000110010
01001100101111111111
11101011111111111010
11000000100111000111
10011101011101101101
11101111111111101110
01001000111000111001
01100001101110110110
10110010100010110110
10011101111101011110
00001001001000011011
00100101010011101011
11110100101111011111
01111100001001001010
11000010111111100110
11110011101010000011
01111000100100000011
00110001001000010111
00001110100001100000
00000100101110001100
11101111111110111111
00110000110010100101
11101110001110100110
00011010101110110011
10000000011000000101
11110011111111000010
10010111110110111111
00000111101011011111
00101101111111111011
11101101011000000110
10000111110101100100
01010101010010111110
00100110101100100011
01100101101111111111
10000110011010010010
00100001000010100010
11001001011100010010
00001000001000011101
01101100110011110101
10001000101011001110
10001001011100111101
00000001000010101111
10000000000110010101
11011100111110000010
11001111001110101101
10001100001000111110
11001000011100111111
00010001111010000000
00011101000011000110
10011101011010111010
10110000110010101100
10000100000011100000
00010001000001001101
01000010101010101111
01110001001111111011
10011000011000000110
10111000111110010001
01111011111001000100
11110101101101110111
00001011111010010010
10100000011111111111
11001111000000101011
10011110110010101011
01100100111100001000
01000100001010100001
01011101101110111011
10101101011010110010
11111111101111111111
01000001001111111101
10100001011100000000
11011010111101101111
10110001011000110101
11011010010111101110
11100001001110101011
01011111111111111010
00110000001011111101
10001000000000111110
00001000101110010101
10001101111101111010
00100100001010001110
10011011101101110111
11010100101000010000
01000010011111110111
01001000110011100100
11101001110010111100
00010101001010110110
10100100111010010110
10111010110111110110
01011101111111111111
10111101010100110011
01101101101000000000
00000100010000111001
01001111101001100101
10111101011010100110
01001111011110101111
10111101001010101110
10011001010011010010
01100110011011001010
00101100110011001111
10110101011100011011
00011000110111110001
10011111110110100010
11110111100010100000
10111100110011100000
10110111011011111110
00110001001010010110
10000001101111000011
10111101100010110110
01101101010111111101
10110111111111101011
01100001111011010000
01111010101110100100
00000101101111010011
10111011111000111010
11011101000110111111
11001001000110000001
11111101111110001110
11010101111011110101
11010111101111111010
11000111100001001011
