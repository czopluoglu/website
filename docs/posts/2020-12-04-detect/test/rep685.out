-------------------------------------------------------
                  DETECT SUMMARY OUTPUT
-------------------------------------------------------
 
                    Data File Name: C:/Users/cengiz/Desktop/Github/website/_posts/2020-12-04-detect/test/rep685.dat                                         

              Number of Items used:       20

           Number of Items dropped:        0

               Number of Examinees:      250

                 Minimum Number of 
                Examinees per Cell:        2

         Number of Vectors Mutated:        4

      Maximum Number of Dimensions:       12

                Randomization Seed:    99991

   Minimum percentage of examinees
         used after deleting cells
     having less than  2 examinees:    98.80

   Average percentage of examinees
         used after deleting cells
     having less than  2 examinees:    99.62
 
-------------------------------------------------------

 NUMBER OF DIMENSIONS THAT MAXIMIZE DETECT:  4
 
  Exploratory DETECT Statistics:

              Maximum DETECT value:   0.7695
                   IDN index value:   0.7421
                           Ratio r:   0.6762

 PARTITION WITH MAXIMUM DETECT VALUE:

    1    2    1    1    2    1    2    1    2    1
    3    3    4    3    4    3    3    4    3    3

 CLUSTER MEMBERSHIPS:

   -----------CLUSTER  1-------------

    1    3    4    6    8   10

   -----------CLUSTER  2-------------

    2    5    7    9

   -----------CLUSTER  3-------------

   11   12   14   16   17   19   20

   -----------CLUSTER  4-------------

   13   15   18
 
   ----------------------------------
 
  Covariance Sign Pattern Matrix:
   d-++-+-++------+----
   -d+++++-+-+---------
   ++d-++-------+-+----
   ++-d+-+-+---+-------
   -+++d+-+--+----+----
   +++-+d-++---------+-
   -+-+--d++++++---+++-
   +---+++d+---------++
   ++-+-+++d++----+--+-
   ------+-+d-+----+---
   -+--+-+-+-d+++-++---
   ------+--++d+++++---
   ---+--+---++d++-++-+
   --+-------+++d-+++++
   -----------++-d++-++
   +-+-+---+-++-++d-+--
   ------+--++++++-d-++
   ------+-----++-+-d++
   -----++++----++-++d+
   -------+----+++-+++d
 
 No cross validation for this DETECT run
